icd/api/include/vk_utils.h

```diff
@@ -131,12 +131,8 @@ typedef VkAccessFlags2KHR        AccessFlags;
 namespace utils
 {
 
-// =====================================================================================================================
-// Get time in nano seconds
-VK_INLINE uint64_t GetTimeNano()
-{
-    return (static_cast<Pal::uint64>(Util::GetPerfCpuTime()) * NANOSECONDS_IN_A_SECOND) /
-        static_cast<Pal::uint64>(Util::GetPerfFrequency());
+VK_INLINE uint64_t TicksToNano(uint64_t ticks) {
```

Comment:
The opening brace should be on its own line.  XGL is supposed to follow the PAL coding standards.